### PR TITLE
Fallback to super-language before trying 2ndary language

### PR DESCRIPTION
**Pull request by humdingerb**
Seems to work fine.
@janus2 , please review if I did good.

### CODE REVIEW

File: source/Tipster.cpp [hunk 1]
```diff
@@ -480,9 +480,17 @@ Tipster::GetTipsFile()
 	if (roster->GetPreferredLanguages(&message) == B_OK) {
 		const char *language;
 
-		for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++)
+		for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++) {
```
Comment (janus2) [inline source/Tipster.cpp]: for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++) 
			if (getLocalTipsFile(ref, language) 
				|| getLocalTipsFile(ref, BString(language, BString(language).FindFirst("_"))))
				return ref;

**Comment by janus2** [comment]
It's good or if you want a compact solution I have added one as a review

File: source/Tipster.cpp (hunk 1 above)
Comment (janus2) [inline source/Tipster.cpp]: for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++) 
			if (getLocalTipsFile(ref, language) 
				|| getLocalTipsFile(ref, BString(language, BString(language).FindFirst("_"))))
				return ref;

Comment (janus2) [reply source/Tipster.cpp]: Probably yours is more efficient...

Comment (janus2) [reply source/Tipster.cpp]: Or this to remove the tmp BString:
				`BString superLanguage(language);`
				`superLanguage.Truncate(superLanguage.FindFirst("_"));`

**Comment by janus2** [comment]
When hrev51858 is out Tipster will stop work. I will patch the app as soon as the build is out.

**Comment by humdingerb** [comment]
Updated the PR.
I don't think efficiency isn't really an issue as the loop is run at max a couple of times on startup.  I kept the "if...else", because I think it's more easily read.
I do like the "BString(language, BString(language).FindFirst("_")))". Clever!

Merge?

**Comment by janus2** [comment]
Why not?

**Pull request merged**